configsuite/schema.py

```diff
@@ -25,11 +25,29 @@
 from configsuite import types
 
 
+@configsuite.validator_msg("AllowNone can only be used for BasicType")
+def _check_allownone_type(schema_level):
```

Comment:
sorry for being a bit to pedantic here, but it bothers me the inconsistent spelling of allow_none and allownone. would prefer _check_allow_none_type .

Reply:
There is a difference between allowing none and checking the `AllowNone` field. To me `_check_allownone_type` sounds ok, but if you prefer `_check_allow_none_type` is also good.

Reply:
i got your point - leave it as it is :-)